server/utils/listmonk-requests.ts

```diff
@@ -0,0 +1,63 @@
+const runtimeConfig = useRuntimeConfig()
+
```

Comment:
please reference the API documentation mentioning these endpoints and what exactly you are achieving with this implementation